atlas/AtlasMl/atlasml/ml/pipeline_workflows.py

```
@@ -0,0 +1,353 @@
+            competency_id = competency[0]["properties"]["competency_id"]
+            properties = {
+                "exercise_id": exercise["properties"]["exercise_id"],
+                "description": exercise["properties"]["description"],
+                "competency_ids": [competency_id]
+            }
+            self.weaviate_client.update_property_by_id(CollectionNames.EXERCISE.value, exercise["id"], properties)
+
+        return
+
+
+    def newTextPipeline(self, text: str) -> Competency:
```

Ruff (0.12.2)

Function name `newTextPipeline` should be lowercase

(N802)